From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Xiaohan Wang <xhwang@chromium.org>
Date: Tue, 29 Sep 2020 23:47:31 +0000
Subject: Fix HighBD config on Windows ARM64

In the last CL, "cpu_arch_full" was set to "arm64-highbd" for Windows
ARM64, which seems not needed because "arm64 windows has it's own folder
for vp9_rtcd.h etc". This CL revert that part of the change so Windows
restores to the previous behavior.

Tbr: jzern@google.com
Bug: 1133301
Change-Id: Iba23fe496841965f491704d5a503b388c1859b7c
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2437745
Reviewed-by: Xiaohan Wang <xhwang@chromium.org>
Reviewed-by: James Zern <jzern@google.com>
Commit-Queue: Xiaohan Wang <xhwang@chromium.org>
Cr-Commit-Position: refs/heads/master@{#811881}

diff --git a/third_party/libvpx/BUILD.gn b/third_party/libvpx/BUILD.gn
index 6d98144d94f534dd28a4d39f67483062d7450343..df54bc90775be227ef18d83451156e94f681c95a 100644
--- a/third_party/libvpx/BUILD.gn
+++ b/third_party/libvpx/BUILD.gn
@@ -29,7 +29,7 @@ if (current_cpu == "x86") {
     cpu_arch_full = "arm"
   }
 } else if (current_cpu == "arm64") {
-  if (is_chromeos || is_win || is_mac) {
+  if (is_chromeos || is_mac) {
     cpu_arch_full = "arm64-highbd"
   } else {
     cpu_arch_full = current_cpu
